tests/kits/test/src/integration.test.js

```diff
@@ -20,13 +19,14 @@ function runIntegrationTest({ kit }, action) {
   before('cleaning up project folder', cleanup);
 
   it('unpack kit', function() {
-    // have to replace oz with local version so we test current build
     run(`node ../../../packages/cli/lib/bin/oz-cli.js unpack ${kit}`);
   });
 
   // have to replace oz with local version so we test current build
   it('replace oz with local version', function() {
-    run(`${findUp.sync('node_modules/.bin/lerna')} bootstrap --scope=tests-cli-kits --scope="@openzeppelin/*"`);
+    const version = require('../package.json').version;
```

Comment:
Is this a leftover from the yarn PR?

Reply:
I would have expected this to be changed on that PR, but I did it here as I was reviewing problems with compilation at this step. I'm not sure it's necessary as part of the upgrade, but I think it's nice to have everything on the same toolchain.